rero/rero-ils
# acquisition: create library order

* Adds order resource for acquisition.
* Adds permissions for order resource.
* Adds order line resource for acquisition.
* Adds library and order_status facets for orders.
* Adds sort to order lines.
* Creates tests.
* Creates fixtures for tests.
* Fixes fuzzy translations for vendors.

Co-Authored-by: Lauren-D <laurent.dubois@itld-solutions.be>

## Why are you opening this PR?

- Implement order and order line resource

## How to test?

- Use rero/rero-ils-ui/pull/106
- Check order entry in acquisition menu

## Code review check list
